fe/fe-core/src/main/java/com/starrocks/connector/parser/trino/Trino2SRFunctionCallTransformer.java

```diff
@@ -144,6 +145,23 @@ private static void registerDateFunctionTransformer() {
         registerFunctionTransformer("to_unixtime", 1, "unix_timestamp",
                 List.of(Expr.class));
 
+        //from_unixtime -> convert_tz
+        registerFunctionTransformer("from_unixtime", 2,
```

Comment:
from_unixtime support 1,2,3 arguments, not 2 arguments.

Reply:
I have added support for 1,2 and 3 arguments. Please see latest commit.